### ISSUE TITLE
Instability for Boundary Fuel Model

### ISSUE THREAD

**Issue by mcgratta** [firemodels/fds#11825]
This case 
[test.txt](https://github.com/firemodels/fds/files/11465942/test.txt)
has similar properties to CSIRO Case F19. It runs only because I increased the thermal conductivity of everything to a high value, but even then it is very unstable. The char forms and is consumed very quickly leading to wild temperature swings.

**Comment by ericvmueller** [comment]
With the changes I have [here](https://github.com/firemodels/fds/compare/master...ericvmueller:fds:bfm) to update the definition of `RHO_S` and scale conductivity by `PACKING_RATIO`, the example you have works when shrinking is allowed. 

Case_F19_BFM_crude from CSIRO also works, but Case_C064_BFM_crude fails. However, if I use the little bluestem grass pyrolysis model rather than the generic one then both appear to work, albeit with somewhat high surface temperatures...

Another option might be to continue to enforce some `MINIMUM_BURNOUT_TIME` for now to prevent things becoming unstable.

**Comment by ericvmueller** [comment]
The test case works with the above commits ...but we may want to leave this issue open while I explore removing `MINIMUM_BURNOUT_TIME` from validation cases

**Comment by mcgratta** [comment]
Can this issue be closed?

**Comment by ericvmueller** [comment]
Yup - I'm not aware of instabilities in any of the boundary fuel test cases as they are now
